ldk/javascript/src/user/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
We added basic unit tests for all of the other aptitudes. I'd vote to keep this aptitude consistent (even though they are just asserting that calls are being made to our `oliveHelps` reference).

Reply:
Not a fan.  The tests are extremely low value (imo), don't document anything, and probably add more confusion and build time.  But I'll do it if the team values consistency.